src/site/paragraphs/collapsible_panel.drupal.liquid

```diff
@@ -26,19 +26,27 @@
 {% if entity.fieldCollapsiblePanelBordered %}
     {% assign accordionClass = 'usa-accordion-bordered' %}
 {% endif %}
+
+{% assign ariaExpanded = entity.fieldCollapsiblePanelExpand %}
+
+{% assign ariaHidden = true %}
+{% if ariaExpanded == true  %}
+  {% assign ariaHidden = false %}
+{% endif %}
+
 <div class="{{ accordionClass }}" data-multiselectable="{{ entity.fieldCollapsiblePanelMulti }}">
     <ul class="usa-unstyled-list">
         {% for accordionItem in entity.fieldVaParagraphs %}
             {% assign item = accordionItem.entity %}
             <li>
                 <button
                         class="usa-accordion-button usa-button-unstyled"
-                        aria-expanded="{{ entity.fieldCollapsiblePanelExpand }}"
+                        aria-expanded="{{ ariaExpanded }}"
```

Comment:
` aria-expanded` has to be `true` or `false`. Prior to this change, it was showing up as the empty string instead of `false` 🤷‍♂